.travis.yml

```diff
@@ -3,6 +3,7 @@ branches:
     only:
         - master
         - dev
+        - regressors
```

Comment:
Do we need this? Since it's a PR into dev, it should trigger the CI builds, or does it not? Anyways, we won't need it after merging this branch.

Reply:
Removed and committed, thanks.